client/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { NavLink, Route } from 'react-router-dom';
+import './App.css';
+
+import Home from './components/Home.js';
+import SignUp from './components/SignUp.js';
+import SignIn from './components/SignIn.js';
+import Users from './components/Users.js';
+
+class App extends Component {
+  signOut() {
+    localStorage.removeItem('jwt');
+  } ;
+
+  render() {
+    return (
+      <div className="App">
+        <nav>
+          <NavLink to='/'>
+           [ Home ] 
+          </NavLink>
+          <NavLink to='/signup'>
+           [ Sign up ] 
+          </NavLink>
+          <NavLink to='/signin'>
+           [ Sign in ] 
+          </NavLink>
+          <NavLink to='/users' onClick={this.redirectToUsers}>
```

Comment:
For redirecting, I had to do some digging and came up with using a <Redirect />! Great work on completing this portion.